content/manuals/build/bake/targets.md

```diff
@@ -96,6 +96,42 @@ command.
 $ docker buildx bake all
 ```
 
+## Pattern matching for targets and groups
+
+Bake supports shell-style wildcard patterns when specifying target or grouped targets.
+This makes it easier to build multiple targets without listing each one explicitly.
+
+Supported patterns:
+
+- `*` matches any sequence of characters
+- `?` matches any single character
+- `[abc]` matches any character in brackets
+
+Examples: 
+
+```console
+# Match all targets starting with 'foo-'
+$ docker buildx bake foo-*
+
+# Match all targets
+$ docker buildx bake *
+
+# Matches: foo-baz, foo-caz, foo-daz, etc.
+$ docker buildx bake "foo-?az"
+
+# Matches: foo-bar, boo-bar
+$ docker buildx bake "[fb]oo-bar"
+
+# Matches: mtx-a-b-d, mtx-a-b-e, mtx-a-b-f
+$ docker buildx bake "mtx-a-b-*"
+``` 
+
+You can also combine multiple patters:
+
+```console
+$ docker buildx bake "*" "tests"
```

Comment:
`tests` would not be necessary here as we glob everything already. Maybe instead:

```suggestion
$ docker buildx bake "foo*" "tests"
```